src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,113 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+/**
+ * This class encapsulates the command to add tasks. The supported
+ * Task types are Deadline, Event and Todo.
+ */
+public class AddCommand extends Command {
+
+    private String taskType;
+
+    /**
+     * Constructor for AddCommand.
+     * @param userInput String containing the user input
+     * @param taskType String identifying type of task to be added

```

Comment:
The full stop thing I mentioned earlier, very minor detail so I won't mention it anymore